packages/venia-concept/venia-upward.yml

```diff
@@ -190,7 +190,7 @@ cmsPageResult:
 urlKey:
   when:
     - matches: request.url.pathname
-      pattern: '^/(?:(.*)\.html)?'
+      pattern: '^/(?:(.*))?'
```

Comment:
This is the right idea (until we have the store config introspection requested in https://github.com/magento/graphql-ce/issues/535) but it needs to go further down in the stack. Since CMS pages have no suffixes, they now live in the same namespace as some of our hardcoded routes, like `checkout`. Those hardcoded routes need to resolve first--otherwise, an administrator could accidentally break their store by writing an SEO Friendly URL that conflicts with a builtin PWA route, AND, UPWARD would place the urlResolver call unnecessarily for every single request.